### PR TITLE
WarpX 22.04+: Fix RZ Test

### PR DESCRIPTION
**Pull request by ax3l**
In WarpX 22.04, we introduced the openPMD `thetaMode` for fields in RZ geometry. That means we need to name the fields differently than the reconstructured Cartesian slice that we default to in plotfiles.

cc @RTSandberg

### CODE REVIEW

**Review by alalazo — APPROVED** [review]
@ax3l Feel free to merge at you will. Not merging right now in case you want to wait for some other maintainers to review.

**Comment by ax3l** [comment]
Thx, no all good to go :)

**Pull request merged**
